src/js/control-bar/progress-control/progress-control.js

```diff
@@ -28,6 +29,9 @@ class ProgressControl extends Component {
     super(player, options);
     this.handleMouseMove = Fn.throttle(Fn.bind(this, this.handleMouseMove), 25);
     this.on(this.el_, 'mousemove', this.handleMouseMove);
+
+    this.throttledHandleMouseMove = throttle(bind(this, this.handleMouseSeek), 25);
```

Comment:
maybe it would be a good idea to add a doc comment for this method, or even add it to the class and have it do the same thing it does now.